File: python/Tiket.py
class Tiket:
    def __init__(self, idTiket, nomorKursi, harga, statusTiket):
        self.__idTiket = str(idTiket)
        self.__nomorKursi = str(nomorKursi)
        self.__harga = int(harga)
        self.__statusTiket = str(statusTiket)

    def getIdTiket(self):
        return self.__idTiket

    def getNomorKursi(self):
        return self.__nomorKursi

    def getHarga(self):
        return self.__harga

    def getStatusTiket(self):
        return self.__statusTiket

    def setIdTiket(self, idTiket):
        self.__idTiket = str(idTiket)

    def setNomorKursi(self, nomorKursi):
        self.__nomorKursi = str(nomorKursi)

    def setHarga(self, harga):
        self.__harga = int(harga)

    def setStatusTiket(self, statusTiket):
        self.__statusTiket = str(statusTiket)

File: php/Tiket.php
<?php

class Tiket
{
    private $idTiket;
    private $nomorKursi;
    private $harga;
    private $statusTiket;

    public function __construct(
        $idTiket,
        $nomorKursi,
        $harga,
        $statusTiket
    ) {
        $this->idTiket = $idTiket;
        $this->nomorKursi = $nomorKursi;
        $this->harga = (int) $harga;
        $this->statusTiket = $statusTiket;
    }

    public function getIdTiket()
    {
        return $this->idTiket;
    }

    public function getNomorKursi()
    {
        return $this->nomorKursi;
    }

    public function getHarga()
    {
        return $this->harga;
    }

    public function getStatusTiket()
    {
        return $this->statusTiket;
    }

    public function setIdTiket($idTiket)
    {
        $this->idTiket = $idTiket;
    }

    public function setNomorKursi($nomorKursi)
    {
        $this->nomorKursi = $nomorKursi;
    }

    public function setHarga($harga)
    {
        $this->harga = (int) $harga;
    }

    public function setStatusTiket($statusTiket)
    {
        $this->statusTiket = $statusTiket;
    }
}
?>
